proxies/live/apiproxy/policies/KeyValueMapOperations.GetSecureVariables.xml

```diff
@@ -1,8 +1,4 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- Get any eRS specific Key Value Map parameters -->
```

Comment:
As discussed over slack, given we're not currently using a Key Value Map in Apigee we can probably remove this policy for now.